webpack/ForemanTasks/Components/TaskDetails/__tests__/TaskDetailsActions.test.js

```diff
@@ -1,4 +1,4 @@
-import { testActionSnapshotWithFixtures } from 'react-redux-test-utils';
+import { testActionSnapshotWithFixtures } from '@theforeman/test';
```

Comment:
IMO let's not migrate it yet, we can do it later and move the whole logic from `react-redux-test-utils` to `@theforeman/test`

Reply:
As we discussed offline, IMO `react-redux-test-utils` has a justification to stay standalone for open source sharing